### DISCUSSION TITLE
Retropie Synchronisation

### DISCUSSION THREAD

**Discussion by paulmortlock** [sam-ludlow/mame-ao#35] [Ideas]
Well is there a way to automatically copy across the files that are downloaded to a Retropie.

**Comment by sam-ludlow** [comment]
Hello you can use the import function, MAME-AO will grab anything that's in the current MAME.

https://github.com/sam-ludlow/mame-ao?tab=readme-ov-file#import

**Comment by paulmortlock** [comment]
I can use the export function to export to the samba share that the retropie creates for managing roms.

But it creates another weird issue on the retropie and may be complicated to explain. Do you have a Raspberry Pi 3 or 4 to check.
